twitter.html

```diff
@@ -0,0 +1,359 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>twitter</title>
+    <link rel="stylesheet" href="twitter.css" />
+  </head>
+  <body>
+    <div class="container">
+      <!-- HEADER SECTION -->
+      <div class="header">
+        <div><h1>Home</h1></div>
+      </div>
+      
```

Comment:
have a look at the formating trick i suggested in the support channel to help you get your code looking nicer

Reply:
hi Phil thanks for your feedback, i thought i did before every push but perhaps I may have missed it on the last push. ill keep in mind. thanks